features/step_definitions/logging.rb

```diff
@@ -959,15 +959,15 @@
 # version: 6.8 or 7.12
 # project_name: where the external ES deployed
 # scheme: http or https
-# transport_ssl_enabled: true or false
+# client_auth: true or false
```

Comment:
Could we explain here?  
client_auth: true or false. When true, client crendential(Eeither certs files or user/passwd or both) must be provided to access ES.

Reply:
OK